drivers/soft_spi/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = soft_spi
```

Comment:
this line is not needed, the module name is deducted from the path automatically...